.github/workflows/deploy.yaml

```diff
@@ -25,27 +25,8 @@ jobs:
 
             - name: Build and push image
               run: |
-                # Cache multi-stage builds: https://pythonspeed.com/articles/faster-multi-stage-builds/
-
-                docker pull rector/getrector.org:base || true
-                docker pull rector/getrector.org:node-build || true
                 docker pull rector/getrector.org:latest || true
-
-                docker build . --tag rector/getrector.org:base --target base \
-                  --cache-from rector/getrector.org:base
-
-                docker build . --tag rector/getrector.org:node-build --target node-build \
-                  --cache-from rector/getrector.org:base \
-                  --cache-from rector/getrector.org:node-build
-
-                docker build . --tag rector/getrector.org:latest --target production \
-                  --build-arg commitHash=${{ github.sha }} \
-                  --cache-from rector/getrector.org:base \
-                  --cache-from rector/getrector.org:node-build \
-                  --cache-from rector/getrector.org:latest
-
-                docker push rector/getrector.org:base
-                docker push rector/getrector.org:node-build
+                docker build . --tag image --cache-from rector/getrector.com:latest
```

Comment:
@JanMikes I tried to mimic https://github.com/deprecated-packages/cleaningladylist.com/pull/24/files#diff-4f9e38227ed64fefb17f4668a7ac4ab55b6149994d5ac2fd96182d5958479b54
here. Making it simple :) 

What alse should be fixed here? :)